canonical/ubuntu.com
# Build /user-research

## Done

- Builds user-research based on [copydoc](https://docs.google.com/document/d/1gOoOJNnrUVWOhpYlkShq0FAbMqDZsj-CXuOGT_S8Xl4/edit#) and [design](https://www.figma.com/file/l6Hfa6Y9TyVrx6m0XySTKx/UXR-page?node-id=0%3A1)
- Removes styling that was waiting for a vanilla issue to be merged, see diff for more info
## QA

- Go to https://ubuntu-com-12637.demos.haus/user-research and check that the content is the same as in the [copydoc](https://docs.google.com/document/d/1gOoOJNnrUVWOhpYlkShq0FAbMqDZsj-CXuOGT_S8Xl4/edit#)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1481
